cylc/flow/scripts/validate.py

```diff
@@ -80,6 +105,16 @@ def get_option_parser():
         default="live", dest="run_mode",
         choices=['live', 'dummy', 'simulation'])
 
+    parser.add_option(
+        '--reinstall-validate', '--revalidate', '--against-source',
```

Comment:
The one option should be enough:

```suggestion
        '--against-source',
```